torch/csrc/autograd/python_variable.cpp

```diff
@@ -2456,4 +2473,17 @@ c10::Layout concrete_layout_fn(
   return toLayout(out.ptr());
 }
 
+template<const char* func_name, typename... Ts>
+void concrete_trace_cuda(const c10::impl::PyInterpreter*, Ts... args) {
+  pybind11::gil_scoped_acquire gil;
+  at::impl::MaybeSetTLSOnEntryGuard guard;
+
+  try {
+    py::module mod = py::module::import("torch.utils._cuda_trace");
+    py::object hook = mod.attr(func_name).attr("fire_callbacks");
+    hook(args...);
+  }
+  catch (const py::error_already_set& e) {}
```

Comment:
What do you think of printing out something in this case? Silently swallowing errors makes it very hard to debug...

Reply:
As discussed, I've added a comment explaining why I've kept the catch silent.